pod/perlclass.pod

```diff
@@ -260,6 +260,33 @@ context.
 
     scalar $instance->users;
 
+=head3 :writer
+
+A field with a C<:writer> attribute will generate a writer accessor method
+automatically.  The generated method will have a signature that consumes
+exactly one argument, and its body will assign that scalar argument to the
+field, and return the invocant object itself.
```

Comment:
The comma on this line is incorrect and should be removed.

Reply:
Thanks.  Committed to blead in 9e152dfc44.